django-backend/fecfiler/web_services/dot_fec/schema_fields/F1M.json

```diff
@@ -28,8 +31,8 @@
         "path": "form_1m.affiliated_committee_name"
     },
 
-    "I_candidate_id_number": {
-        "path": "form_1m.I_candidate_id_number"
+    "I_candidate_id": {
+        "path": "form_1m.I_candidate_id"
```

Comment:
@dheitzer Are yo sure about this change? In the model I see: I_candidate_id_number

Reply:
@mjtravers nice catch.  I pushed a fix but can only fully test when 1555 goes through.

Reply:
add it to the list of field names we'd love to change if we could change the names in the spec